src/geo/transform.js

```diff
@@ -365,19 +399,25 @@ class Transform {
 
         this._constraining = true;
 
-        let minY, maxY, minX, maxX, sy, sx, x2, y2;
+        let minY = -90;
+        let maxY = 90;
+        let minX = -180;
+        let maxX = 180;
```

Comment:
Flow was complaining about likes like 449 since they could still be uninitialized. It seems a little unexpected different projects would use different min/max values for longitude and latitude.